### PR TITLE
VSL-187: Updated status and actions in Things To Do

### PR DESCRIPTION
**Pull request by KsenijaZivkovic**
## Description

What is done:
- Renamed onConfirm to onExecute
- Renamed statuses and buttons, and added proper Svgs
- Added titles above the things to do table

NOTE: Parts of the text weren't bolded because it would require a lot of text manipulation, which is prone to errors and constant modifications in case of intent changes. This text comes from BE (action view -> intent).

## Demo / Test Result

Screen Shot 2022-09-28 at 3 40 48 PM

Screen Shot 2022-09-28 at 3 41 06 PM

Screen Shot 2022-09-28 at 3 41 33 PM

Screen Shot 2022-09-28 at 3 41 55 PM

### CODE REVIEW

**Comment by mannynotfound** [comment]
empty/mobile views are a bit off:

image

image

File: packages/client/src/components/Actions/ActionsList.js
Comment (linxiao-li) [inline packages/client/src/components/Actions/ActionsList.js:58]:
```suggestion
        className="p-0 is-flex is-justify-content-center is-align-items-center has-text-grey"
```

File: packages/client/src/components/Actions/ActionsList.js
Comment (linxiao-li) [inline packages/client/src/components/Actions/ActionsList.js:82]:
```suggestion
          className="p-5 m-0 columns is-align-items-center"
```

File: packages/client/src/library/Svg/Confirm.js
Comment (linxiao-li) [inline packages/client/src/library/Svg/Confirm.js:1]: Can we rename this svg to describe the icon (for example, CircleCheck)? It's easier to be referenced by other components. Same with Execute

Comment (KsenijaZivkovic) [reply packages/client/src/library/Svg/Confirm.js:1]: Not sure which other components would use them, it looks like it would be used only for confirming/executing, plus these icons might be changed so I would leave the name as-is for now, and change it when we get the final design.

Comment (KsenijaZivkovic) [reply packages/client/src/library/Svg/Confirm.js]: Updated the names to match the look of the icon!

File: packages/client/src/components/Actions/ActionsList.js
Comment (KsenijaZivkovic) [reply packages/client/src/components/Actions/ActionsList.js]: Fixed

File: packages/client/src/components/Actions/ActionsList.js
Comment (KsenijaZivkovic) [reply packages/client/src/components/Actions/ActionsList.js]: Fixed

**Comment by KsenijaZivkovic** [comment]
> empty/mobile views are a bit off:
> 
> image
> 
> image

@mannynotfound fixed this too!

File: packages/client/src/components/Actions/ActionsList.js
Comment (mannynotfound) [inline packages/client/src/components/Actions/ActionsList.js:69]: can we make this a `table` and use the markup structure from https://github.com/DapperCollectives/VESSEL/blob/main/packages/client/src/components/TransactionTable.js#L73-L93 ?

since both of these are on the same page they should be nearly identical in spacing/mobile 

image

image

Comment (KsenijaZivkovic) [reply packages/client/src/components/Actions/ActionsList.js:69]: In case of mobile - confirm button should be below, is that possible to achieve with tables?

Comment (mannynotfound) [reply packages/client/src/components/Actions/ActionsList.js:69]: @KsenijaZivkovic might not be - you might just have to make the desktop confirm `is-hidden-mobile` and then have a "mobile" confirm button that is just a full table row only visible on mobile

File: packages/client/src/components/Actions/ActionsListTable.js
Comment (linxiao-li) [inline packages/client/src/components/Actions/ActionsListTable.js:61]: Line 61 62 63 should be placed on top before they are used. ESlint should be able to catch that, let me know if you are not seeing the warning from ESLint

File: packages/client/src/components/Actions/ActionsListTable.js
Comment (linxiao-li) [inline packages/client/src/components/Actions/ActionsListTable.js:71]: what does the key property do?

File: packages/client/src/components/Actions/ActionsListTable.js
Comment (linxiao-li) [inline packages/client/src/components/Actions/ActionsListTable.js:66]: Looks like we can remove this empty div and just return tr

File: packages/client/src/pages/Safe.js
Comment (linxiao-li) [inline packages/client/src/pages/Safe.js:46]: missing type property

File: packages/client/src/components/Actions/ActionsListTable.js
Comment (KsenijaZivkovic) [reply packages/client/src/components/Actions/ActionsListTable.js:71]: There was an issue - color didn't change for the svg once signer approved the action, so when key is changed the color is updated as well

File: packages/client/src/components/Actions/ActionsListTable.js
Comment (KsenijaZivkovic) [reply packages/client/src/components/Actions/ActionsListTable.js:66]: There are two <tr> tags inside, so it should stay

File: packages/client/src/components/Actions/ActionsListTable.js
Comment (KsenijaZivkovic) [reply packages/client/src/components/Actions/ActionsListTable.js:5]: Doesn't work :(

File: packages/client/src/components/Actions/ActionsListTable.js
Comment (KsenijaZivkovic) [reply packages/client/src/components/Actions/ActionsListTable.js]: Updated!

File: packages/client/src/components/Actions/ActionsListTable.js
Comment (KsenijaZivkovic) [reply packages/client/src/components/Actions/ActionsListTable.js]: Fixed

File: packages/client/src/components/Actions/ActionsListTable.js
Comment (linxiao-li) [reply packages/client/src/components/Actions/ActionsListTable.js:30]: Got it, I'll fix the Svg loader

File: packages/client/src/components/Actions/ActionsListTable.js
Comment (linxiao-li) [inline packages/client/src/components/Actions/ActionsListTable.js:109]: Looks like we don't need the template literal here

**Review by linxiao-li — APPROVED** [review]
LGTM overall!

File: packages/client/src/components/Actions/ActionsListTable.js
Comment (KsenijaZivkovic) [reply packages/client/src/components/Actions/ActionsListTable.js:25]: The second row is visible only in mobile view, to display the button, and in the first row the button is visible in desktop view

Comment (KsenijaZivkovic) [reply packages/client/src/components/Actions/ActionsListTable.js:25]: I used Manny's idea from this comment https://github.com/DapperCollectives/VESSEL/pull/118#discussion_r982906870

File: packages/client/src/components/Actions/ActionsListTable.js
Comment (KsenijaZivkovic) [reply packages/client/src/components/Actions/ActionsListTable.js]: Fixed!

**Review by mannynotfound — APPROVED** [review]

**Comment by almendili** [comment]
Front-end feedback:
- Can we use title case for "Transaction history" -> "Transaction History"
- Can we add a grey line between items in Things To Do and Transaction History
- Can we reduce the padding for each line. Right I think it's 1rem. Could we go to .5rem

Otherwise, everything looks great!

**Pull request merged**
